docs/datamaps/structure.md

```diff
@@ -0,0 +1,103 @@
+# JSON Structure
+For the purposes of this page, we will use a data map which is an object with two float keys: `amount` and `chance` as an example. The codec for that object can be found [here](./index.md#registration).
+
+## Location
+Data maps are loaded from a JSON file located at `:mapNamespace/data_maps/:registryNamespace/:registryPath/:mapPath.json`, where:
```

Comment:
As above, I find the colons confusing. Just do `mapNamespace/data_maps/registryNamespace/registryPath/mapPath.json` and it should do.